### ISSUE TITLE
Help for a consulting

### ISSUE THREAD

**Issue by estebanmax** [stuarta0/blender-photogrammetry#15]
Hello, really great addon.
As I also wrote on youtube, I tried to use it but got some errors in Blender (v. 2.91 on Windows).
The point is that I really need it for my projects, I've been looking for something link that for a long time.
Sometimes I used PfTrack to do that but the workflow is much more difficult.
So I would like to agree with you also for a paid consultation in order to solve the issue.
Thank you a lot.
Stefano

**Comment by stuarta0** [comment]
Hi Stefano,

So this was your stack trace?
```
Traceback (most recent call last):
  File "/Applications/Blender.app/Contents/Resources/2.90/scripts/modules/addon_utils.py", line 351, in enable
    mod = __import__(module_name)
ModuleNotFoundError: No module named 'blender-photogrammetry-1'
```

Looks like you're on MacOS which is unsupported. Since the addon's primary purpose is dense reconstruction, and most software use CUDA to achieve it, and Mac's generally don't have Nvidia cards, it's not worth the effort. Regardless, even if one of the libraries didn't use CUDA (e.g. PMVS uses CPU), they generally don't have a Mac build, so I haven't included it.

If you still wanted to use it for converting between photogrammetry formats, the actual problem is the version number in the ZIP. Github automatically appends it if you're downloading the source code from the v1.1 tag. Instead, go to the master branch and download a ZIP. Github will name this 'blender-photogrammetry-master' and Blender won't choke when trying to import it.

**Comment by stuarta0** [comment]
Module lookup fixed in 8f6039a8714dcb8b436739eedde8dc69324fa9ea
